app/delivery/send_to_providers.py

```diff
@@ -216,12 +217,6 @@ def provider_to_use(notification_type, notification_id, international=False, sen
     return clients.get_client_by_name_and_type(active_providers_in_order[0].identifier, notification_type)
 
 
-def get_logo_url(base_url, logo_file):
-    bucket = current_app.config['ASSET_UPLOAD_BUCKET_NAME']
-    domain = current_app.config['ASSET_DOMAIN']
-    return "https://{}.{}/{}".format(bucket, domain, logo_file)
```

Comment:
Moved to utils as it's now used in 2 places. The `base_url` parameter was not used.